workflow.txt

```diff
@@ -1,10 +1,6 @@
-git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
+(make changes)
 git add .
-git pull upstream master
-git merge my-solution
-fork on github
+git commit -m "(write message)"
 git checkout master
-git push origin my-solution
+git merge my-solution
```

Comment:
This workflow works for merging content from a feature branch to a master branch locally. Although the intent with this question is for you to push a feature branch up to github and then submit a pull request for it (i.e., you wouldn't have to merge anything locally).